server/controllers/wazuh-reporting.ts

```diff
@@ -1906,7 +1906,7 @@ export class WazuhReportingCtrl {
             ]
           }
         },
-        {
+        agentOs === 'windows' && {
```

Comment:
this is a bad implementation, is not a good practice using a ternary in an array definition 
please use a different approach, maybe inserting after the declaration of the array with a conditional 

also, check if this endpoint is used for another SO but windows

Reply:
I change it using your suggestions. Thanks you!